packages/ag-charts-community/src/chart/series/cartesian/bubbleSeriesModule.ts

```diff
@@ -22,6 +23,39 @@ export const BubbleSeriesModule: SeriesModule<'bubble'> = {
 
     identifier: 'bubble',
     moduleFactory: (ctx) => new BubbleSeries(ctx),
+    predictAxis: (
+        direction: ChartAxisDirection,
+        datum: DatumDefault,
+        seriesOptions: AgBubbleSeriesOptions | AgScatterSeriesOptions
+    ) => {
+        if (direction !== ChartAxisDirection.X && direction !== ChartAxisDirection.Y) return;
+        if (!isObject(datum)) return;
+
+        const key = direction === ChartAxisDirection.X ? seriesOptions.xKey : seriesOptions.yKey;
+        if (!(key in datum)) return;
+
+        const value = datum[key];
+        const position = direction === ChartAxisDirection.X ? CARTESIAN_POSITION.BOTTOM : CARTESIAN_POSITION.LEFT;
+
+        if (typeof value === 'number') {
+            return {
+                type: CARTESIAN_AXIS_TYPE.NUMBER,
+                position,
+            };
+        }
+
+        if (value instanceof Date) {
+            return {
+                type: CARTESIAN_AXIS_TYPE.UNIT_TIME,
```

Comment:
Are we sure unit time here? I'd have thought continuous time

Reply:
Fair question, there's been some discussion and it's not necessarily finalised, check the ticket ACs.